src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -64,6 +64,34 @@ public class PublicationService : IPublicationService
             _themeCacheService = themeCacheService;
         }
 
+        public async Task<Either<ActionResult, List<PublicationViewModel>>> GetPublicationsByTopic(
+            bool permissions,
+            Guid? topicId)
```

Comment:
Would give defaults to both of these parameters `permissions = false`, `topicId = null`

Reply:
> I've left a comment above to say two separate methods GetAllPublications(bool permissions = false) and GetPublicationsByTopic(Guid topicId, bool permissions = false) might help make the implementation clearer and make the testing clearer too, but happy to keep it as one method also.

As mentioned in another comment, discussed this on Slack with Ben and Nick and we've decided to keep things as they are.